### DISCUSSION TITLE
How to remove mp4 file on user-images site

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31336] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I have uploaded below video file on README by using Drag and Drop.
So, the file was placed in user-images site. I want to replace the file,
however I do not know how to remove this file.

How can I remove this file?

https://user-images.githubusercontent.com/23236771/187073333-32362911-8dc8-452e-9da1-b285aa184e52.mp4

**Comment by mbiesiad** [comment, marked as answer]
GitHub Support can help with deleting a specific file: https://support.github.com/

Best wishes,

**Reply by ghost** [reply]
Thank you for your quick response.
I lost My Tickets link on GitHub support.

I will post my question to the support.

**Reply by mbiesiad** [reply]
My pleasure! 🎉
